FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/Jobs/Models/FhirToDataLakeOrchestratorJobStatus.cs

```diff
@@ -0,0 +1,88 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Health.AnalyticsConnector.Core.Jobs.Models
+{
+    public class FhirToDataLakeOrchestratorJobStatus
+    {
+        /// <summary>
+        /// Job start time
+        /// </summary>
+        public DateTimeOffset StartTime { get; set; } = DateTimeOffset.UtcNow;
+
+        /// <summary>
+        /// Job complete time
+        /// </summary>
+        public DateTimeOffset? CompleteTime { get; set; }
+
+        /// <summary>
+        /// Created job count
+        /// </summary>
+        public long CreatedJobCount { get; set; }
+
+        /// <summary>
+        /// Completed job count
+        /// </summary>
+        public long CompletedJobCount { get; set; }
+
+        /// <summary>
+        /// Processing job input data that are in the process of submitting, used for system scope
+        /// </summary>
+        public string[] SubmittingProcessingJob { get; set; } = null;
+
+        /// <summary>
+        /// Submitted timestamp for each resource type, used for system scope
+        /// </summary>
+        public Dictionary<string, DateTimeOffset> SubmittedResourceTimestamps { get; set; } = new Dictionary<string, DateTimeOffset>();
```

Comment:
Committed?

Reply:
Updated name to Committed and ToBeCommitted.